app/MMB/src/navigation/MMBScreen.js

```diff
@@ -16,16 +18,105 @@ type Props = {|
   |},
 |};
 
-export default class MMBScreen extends React.Component<Props> {
-  static navigationOptions = () => ({
-    title: 'Manage my bookings',
-  });
-
-  render = () => (
-    <Layout
-      menuComponent={<MainMenu />}
-      // containerComponent={<HelpSubmenu />}
-      containerComponent={<OtherSubmenu />}
-    />
-  );
+type State = {|
+  activeContainerComponent: string,
+|};
+
+export const MenuComponents = {
+  'mmb.help': {
+    screen: HelpSubmenu,
+    headerTitle: function HelpSubmenuHeaderTitle() {
```

Comment:
Any reason for not using arrow function here?

Reply:
Every component (function) must have a name.